### PR TITLE
Finalize Swift debug info before running Clang codegen, because it ma… 

### PR DESCRIPTION
**Pull request by adrian-prantl**
…y delete

the llvm module.

rdar://128309024

### CODE REVIEW

**Comment by adrian-prantl** [comment]
@swift-ci test

**Review by felipepiovezan — APPROVED** [review]

File: test/DebugInfo/BridgingHeaderPCH.swift [hunk 1]
```diff
@@ -5,8 +5,6 @@
 
 // CHECK: !DICompileUnit(language: DW_LANG_Swift
 // CHECK: !DICompileUnit(language: DW_LANG_{{ObjC|C99|C11}},
-// CHECK: !DICompileUnit(language: DW_LANG_{{ObjC|C99|C11}},
-// CHECK-SAME:           splitDebugFilename: "{{.*}}.pch"
-// CHECK-SAME:           dwoId:
+// CHECK: !DICompileUnit(language: DW_LANG_{{ObjC|C99|C11}}{{.*}}splitDebugFilename: "{{.*}}.pch",{{.*}}dwoId:
```
Comment (augusto2112) [inline test/DebugInfo/BridgingHeaderPCH.swift:8]: Why did you change these two tests to not use CHECK-SAME?

**Review by augusto2112 — APPROVED** [review]

File: test/DebugInfo/BridgingHeaderPCH.swift (hunk 1 above)
Comment (adrian-prantl) [reply test/DebugInfo/BridgingHeaderPCH.swift:8]: There's another unrelated DICOmpileUnit in-between and the order between that and the one we're checking for has changed.
